frontend/src/component/executiveDashboard/chart-info.ts

```diff
@@ -51,19 +51,19 @@ export const chartInfo = {
             'How the overall health changes over time for the selected projects.',
     },
     averageTimeToProduction: {
```

Comment:
Can we change the name of the key in this object too? Just so it matches the content.